src/lib/parser_txdef.h

```diff
@@ -99,7 +101,21 @@ typedef struct {
 } oasis_tx_t;
 
 typedef struct {
-    oasis_tx_t oasis_tx;
+    publickey_t id;
+    // We are going to read dynamically like for stakingAmendCommissionSchedule
+    publickey_t node;
+    size_t nodes_length;
+    bool allow_entity_signed_nodes;
+} oasis_entity_t;
+
+typedef struct {
+
+    union {
+        oasis_tx_t oasis_tx;
+        oasis_entity_t oasis_entity;
+    };
```

Comment:
we may need to name the union.. maybe something like this?
```
    union {
        oasis_tx_t tx;
        oasis_entity_t entity;
    } oasis;
```
that way later it is possible to MEMZERO the complete union which is safer